ballerina/pipe.bal

```diff
@@ -35,8 +35,9 @@ public isolated class Pipe {
         'class: "io.xlibb.pipe.Pipe",
         paramTypes: ["java.lang.Long"]
     } external;
-    
+
     public isolated function generatePipeWithTimer(int 'limit, Timer timer) = @java:Method {
+        name: "generatePipe",
         'class: "io.xlibb.pipe.Pipe",
         paramTypes: ["java.lang.Long", "io.ballerina.runtime.api.values.BObject"]
```

Comment:
Do we need this?

Reply:
Here, I'm using overloaded methods from the java code. Since we cant use the same name from Ballerina we have to explicitly add the `name` attribute here.